source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ExplicitDimensionTest.cpp

```diff
@@ -531,7 +537,7 @@ namespace AdaptiveCardsSharedModelUnitTest
             std::shared_ptr<ColumnSet> columnSet = std::static_pointer_cast<ColumnSet>(element);
             std::shared_ptr<Column> column = columnSet->GetColumns().front();
             Assert::AreEqual<int>(0, column->GetPixelWidth());
-            Assert::AreEqual<std::string>("20px20", column->GetWidth());
+            Assert::AreEqual<std::string>("stretch", column->GetWidth());
```

Comment:
at LN 524, width is invalid, and in the invalid case, we set it to 'stretch', and that's correct behavior.